app/controllers/user_sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UserSessionsController < ApplicationController
+  
+  skip_before_filter :require_login, except: [:destroy]
+
+  def new
+   @user = User.new
+  end
+
+  def create
+    if @user = login(params[:email], params[:password])
```

Comment:
hmm you're using form_for in your view so you need to call here `params[:session][:email]` and `params[:session][:password]` or use form_tag in your view